js/data.js

```diff
@@ -0,0 +1,50 @@
+import {random} from './util.js';
+import {getRandomItems} from './util.js';
```

Comment:
можно сделать в рамках одного импорта
```suggestion
import {random, getRandomItems} from './util.js';
```